corehq/apps/users/views/mobile/users.py

```diff
@@ -118,6 +120,12 @@ class EditCommCareUserView(BaseEditUserView):
     urlname = "edit_commcare_user"
     page_title = ugettext_noop("Edit Mobile Worker")
 
+    @property
+    def page_name(self):
+        if self.request.is_view_only:
+            return _("Edit Mobile Worker (View Only)")
```

Comment:
Nitpick: "View Mobile Worker" might be clearer.

Reply:
it shows the forms with disabled fields...and it was a request from Dev. but if UAT thinks it should be reworded can update.